l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

```diff
@@ -972,6 +987,38 @@ def test_54_xml_import(self):
         self.assertEqual(invoice.invoice_line_ids[0].price_subtotal, 1.5)
         self.assertEqual(invoice.move_type, "in_refund")
 
+    def test_55_xml_import(self):
+        # Payments may refer to our own bank account (SEPA)
+        to_unlink = []
+        bank = self.env["res.bank"].create(
+            {
+                "bic": "BCITITMM",
+                "name": "Other Bank",
+            }
+        )
+        to_unlink.append(bank)
+        bank_account = self.env["res.partner.bank"].create(
+            {
+                "acc_number": "IT59R0100003228000000000622",
+                "company_id": self.env.user.company_id.id,
+                "partner_id": self.env.user.company_id.partner_id.id,
+            }
+        )
+        to_unlink.append(bank_account)
+        res = self.run_wizard("test55", "IT01234567890_FPR15.xml")
```

Comment:
I test falliscono perché il valore test55 è usato anche in `test_54_xml_in_invoice_registration_date_rec_date` come valore per il wizard a riga 970

Reply:
@matteoopenf hai idea del perchè questo test non falliva sulla 14?

Reply:
sinceramente no, non ho capito perche' fallisce solo di qua, @patrickt-oforce  mi ha dato un input, ma devo tornarci sopra

Reply:
quello che credo e' che nel porting di altri moduli siano stati fixati i test e quindi ci sia per quello l'errore